contracts/src/TeleporterTokenDestination.sol

```diff
@@ -201,10 +209,10 @@ abstract contract TeleporterTokenDestination is
 
         // If the destination blockchain is the source blockchain,
         // no multi-hop is needed. Only the required gas limit for the Teleporter message back to
-        // `sourceBlockchainID` is needed, which is provided by `input.requiredGasLimit`.
+        // {sourceBlockchainID} is needed, which is provided by {input.requiredGasLimit}.
         // Else, there will be a multi-hop transfer to the final destination.
-        // The first hop back to `sourceBlockchainID` requires `MULTI_HOP_REQUIRED_GAS`,
-        // and the second hop to the final destination requires `input.requiredGasLimit`.
+        // The first hop back to {sourceBlockchainID} requires {MULTI_HOP_REQUIRED_GAS},
+        // and the second hop to the final destination requires {input.requiredGasLimit}.
         BridgeMessage memory message;
         uint256 messageRequiredGasLimit = input.requiredGasLimit;
         if (input.destinationBlockchainID == sourceBlockchainID) {
```

Comment:
might want to `require fallback == address(0)` for sending back to sourceBlockChain, similar to how we make sure secondary fee is zero